pages/index.js

```diff
@@ -11,4 +12,4 @@ function Home() {
   );
 }
 
-export default withReduxSaga(Home, { type: 'GET_REPO_SERVER' });
+export default withReduxSaga(configureStore, Home, { type: 'GET_REPO' });
```

Comment:
something like `withReduxSaga(configureStore, actions)(Component)`, sorry for not adding an example before